includes/class-wp-security-bp-users.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/**
+ * The file that defines the Users class
+ *
+ * This class handles all the users requests.
+ *
+ * @link       http://example.com
+ * @since      1.0.0
+ *
+ * @package    wp_security_bp
+ * @subpackage wp_security_bp/includes
+ */
+

```

Comment:
this lines above could be removed, isn't it?

Reply:
This comment is to inform about the *file* and the following comment is to inform about the *class*